hermeto/core/extras/config_show.py

```
@@ -0,0 +1,211 @@
+
+        for field_name, (current_value, default_value) in section_diff.items():
+            yaml_current = _format_yaml_value(current_value)
+            yaml_default = _format_yaml_value(default_value)
+            lines.append(f"  {field_name}: {yaml_current}  # default: {yaml_default}")
+
+        lines.append("")
+
+    return "\n".join(lines)
+
+
+def _format_yaml_value(value: Any) -> str:
```

Ruff (0.16.0)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `value`

(ANN401)